src/measure_theory/measure/measure_space_def.lean

```diff
@@ -225,6 +225,11 @@ lemma measure_Union_null [countable β] {s : β → set α} : (∀ i, μ (s i) =
   μ (⋃ i, s i) = 0 ↔ ∀ i, μ (s i) = 0 :=
 μ.to_outer_measure.Union_null_iff
 
+/-- A version of `measure_Union_null_iff` for unions indexed by Props -/
```

Comment:
```suggestion
/-- A version of `measure_Union_null_iff` for unions indexed by Props
TODO: in the long run it would be better to combine this with `measure_Union_null_iff` by
generalising to `Sort`. -/
```